site/layouts/partials/js-data-attributes.html

```diff
@@ -0,0 +1,23 @@
+<p>
+  Options can be passed via data attributes or JavaScript. For data attributes, append the option name to <code>data-bs-</code>, as in <code>data-bs-animation=""</code>.
+  Make sure to change the case type of the option name from camelcase to kebab-case when passing the options via data attributes.
+  For example, instead of using <code>data-bs-customClass="beautifier"</code>, you cna use <code>data-bs-custom-class="beautifier"</code>.
+</p>
+
+<div class="bd-callout bd-callout-???">
```

Comment:
Decide to leave the muted callout or use a colored one

Reply:
Is there anyway to use Markdown here, or does it have to be HTML? We have a `markdown` shortcode I think if that's of any help. I'm not 100% on how Hugo does anything though lol.

Reply:
Pushed a change as well to do some copyediting. Let me know if it still makes sense.

Reply:
We could do it like the other Markdown partials.

Don't forget to decide on the callout before merging